### DISCUSSION TITLE
I cant turn off Home Meta Space

### DISCUSSION THREAD

**Discussion by ErKaf** [element-hq/element-meta#700] [Ideas]
### Steps to reproduce

You added new Meta spaces in the Sidebar options. I really, really like this update.
I instantly activated “People” and “Rooms outside of a space”. But sadly now I can't deactivate Home.

This makes no sense. I already have my “People” and “Rooms outside of a space” Space now. I don't need the home anymore. I think the other two are way Cleaner. 

#### What did you expect?
To turn of the Home space
#### What happened instead?
Its not clickable 

### Operating system

KDE Neon

### Browser information

I use it in Ferdi https://getferdi.com/ (So I think its chromium)

### URL for webapp

https://app.element.io

### Application version

Element version: 1.10.1 Olm version: 3.2.8

### Homeserver

matrix.org

### Will you send logs?

No

**Comment by SimonBrandner** [comment]
I believe this is inentional

**Comment by ErKaf** [comment]
> I believe this is inentional

Why? It would be nice to turn off the home. It makes no sense to activate the other spaces if you can't turn off the home where everything is again.

**Comment by SimonBrandner** [comment]
> Why? It would be nice to turn off the home

Can't speak on behalf of the design/product team

cc @niquewoodhouse

**Comment by HarHarLinks** [comment]
"Rooms outside of a space" and "Users" would suffice as a valid alternative to "Home". If you have a lot of activity in the former 2, it would be a very valid replacement for "Home" (unless "Home" is supposed to gain another feature, like the FOSDEM client homepage). The big difficulty is representing all that dependency in the UI.

**Comment by JokerGermany** [comment]
Then pls allow to move the Home Meta Space to the bottem...

**Comment by ErKaf** [comment]
Any updates?  Using “People” and “Rooms outside of a space” makes literally no sense if you can't deactivate the Home. Then everything is double...
But I would like to use them because IF the home is gone it's all way better sorted and cleaner.

**Comment by ErKaf** [comment]
What? Why did I close it by writing a comment? That was an accident...

**Comment by t3chguy** [comment]
I'd suggest not using the `Close with comment` button in that case
![image](https://user-images.githubusercontent.com/2403652/172171463-8e9f7dad-245f-45c4-b59d-c7d477a9d523.png)

**Comment by ErKaf** [comment]
I can't tho. This always shows. No option to turn off...

**Comment by t3chguy** [comment]
Just click `Comment` - the big green one

**Comment by kittykat** [comment]
Moving to discussions as this is cross-platform :+1:

**Comment by CyberShadow** [comment]
FWIW, the Element fork SchildiChat allows disabling the Home button.
